### DISCUSSION TITLE
go vulncheck unstable

### DISCUSSION THREAD

**Discussion by SamYuan1990** [sustainable-computing-io/kepler#280] [General]
https://github.com/golang/vuln/commit/249bfe399a0ecaeb34fa99d5fa723548beb659fa

go vulncheck seems unstable for now.
@rootfs, @marceloamaral  FYI

**Comment by SamYuan1990** [comment]
just noticed our go vulncheck job may fails, https://github.com/golang/vuln/search?q=envGOVULNDB&type=code and I noticed some code change is happening, so far the job may fails.

**Comment by marceloamaral** [comment]
Can you share the errors in a [pastbin](https://pastebin.com/) file?

**Reply by SamYuan1990** [reply]
so far it seems been fixed :-)
